CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+# v0.25.0 Release - 3/15/2019
+
+*Note*: This release comes with a config change, use `skaffold fix` to permanently upgrade your config to `v1beta7`, however old versions are now auto-upgraded. 
```

Comment:
i didn't understand what you meant by "permanently upgrade" does that mean, once i upgrade to v1beta7, i can't downgrade?

Reply:
yeah, this is our standard notice - what it means that the versions are auto-upgraded - so you don't have to upgrade yourself but you'll have the warning message every time. If you "permanently upgrade" with `skaffold fix` the upgrade is persisted.

I guess we could rephrase to something like this: to upgrade your skaffold.yaml, use `skaffold fix`. If you don't upgrade, skaffold will auto-upgrade in memory as best it can, and print a warning message.